scripts/plugins/settings.py

```diff
@@ -1,13 +1,13 @@
-import os, sys
-import yaml
+import os
+import sys
 import argparse
+import yaml
 
 from munch import DefaultMunch
 
+
 parser = argparse.ArgumentParser(description="Add BitOps Usage")
-parser.add_argument(
-    "--bitops_config_file", "-c", help="BitOps source usage information here"
-)
+parser.add_argument("--bitops_config_file", "-c", help="BitOps source usage information here")
```

Comment:
Nice. Love this direction 👍

Reply:
We're good as long as it fits the 100 characters' width, otherwise multi-lining it.